packages/angular_devkit/build_webpack/BUILD.bazel

```diff
@@ -105,22 +100,24 @@ jasmine_node_test(
     ],
 )
 
-# @external_begin
```

Comment:
Guessing we to keep the @external_begin & end here as well to not break google3 when this is synced in

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051